CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Prerequisite Skills to Contribute
+- [Git](https://git-scm.com/) 
+- [node.js](https://nodejs.org/en/)
```

Comment:
I think we can safely remove this section.
Git is understood, we need it for any contribution and node js is not always required, the website repo will also contain DevOps roadmap notes which will be just markdown and major part of this project.
So, this section can be a bit misleading. Thoughts?

Reply:
i am removing only node.js

Reply:
+1 to that @anubha-v-ardhan we can remove the section